app/models/concerns/hud_reports/households.rb

```diff
@@ -41,8 +41,8 @@ module HudReports::Households
             # puts 'Household Batch: '
             # puts GetProcessMem.new.inspect
             batch.each do |enrollment|
-              hh[enrollment.household_id] ||= []
-              hh[enrollment.household_id] << {
+              hh[get_hh_id(enrollment)] ||= []
+              hh[get_hh_id(enrollment)] << {
```

Comment:
Looks like this is shared between the DQ report and APR (maybe SPM too?) so the `get_hh_id` should get moved up into this concern.